os-hardening/init.sls

```diff
@@ -1,16 +1,16 @@
-hardening:
-  include:
-    - os-hardening.network
-    - os-hardening.system
+include:
+  - os-hardening.network
+  - os-hardening.system
 
+/etc/security/limits.d/10.disable_coredumps.conf:
   file.managed:
-    - name: /etc/security/limits.d/10.disable_coredumps.conf
```

Comment:
@askholme If would be good to keep the ``name`` parameter as well. Don't remove it